nc/source/alloc/Allocator.h

```diff
@@ -88,10 +88,9 @@ namespace nc::alloc
     template<class T, class U, class TTag, class UTag>
     bool operator!=(const PoolAllocator<T, TTag>& lhs, const PoolAllocator<U, UTag>& rhs) noexcept { return !(lhs == rhs); }
 
-    /** PoolAllocator */
     template<class T, class Tag>
     PoolAllocator<T, Tag>::PoolAllocator()
-        : m_memoryResource{GetMemoryResource<memory_resource_type>()}
+        : m_memoryResource{MemoryResource<memory_resource_type>::Get()}
```

Comment:
Ignore all changes in this folder - maybe a get latest issue? I didn't make these changes